### DISCUSSION TITLE
Migrating openSUSE Leap to SLE will Uyuni survive?

### DISCUSSION THREAD

**Discussion by corosou** [uyuni-project/uyuni#7580] [Q&A]
Hi Community,

I'm a sysadmin for a US gov agency and we've been running Uyuni on openSUSE to manage a few systems, but unfortunately, openSUSE lacks FIPS compliance, and we've been asked to migrate to SLE.

Were I to follow the steps outlined here: https://en.opensuse.org/SDB:How_to_migrate_to_SLE

Would my Uyuni installation survive? And if so, would I still be able to update it? Or is there a way to install/re-install Uyuni on SLE? I only see openSUSE listed in the docs.

Regards

**Comment by mcalmer** [comment]
I never tried this, but if you enable all the required modules you should be able to migrate openSUSE to SLE by keeping Uyuni installed.
There might be problems when we have dependencies which might require opensuse identification packages. If you report them, we can think about to remove or relax these dependencies (if they exist at all)
More trouble could make runtime checks especially of the database version. we just expect often that Uyuni, means openSUSE as OS, but not SLES. We might have conditions where this create conflicts.

**Reply by juliogonzalez** [reply]
In short: it could work, but it's not supported if there are any issues (could be time to have a look at SUSE Manager is the SUSE downstream version of Uyuni, and works on top of SUSE Linux Enterprise)
